micro_sam/sam_annotator/_state.py

```diff
@@ -57,14 +99,16 @@ def initialized_for_interactive_segmentation(self):
     def initialized_for_tracking(self):
         have_current_track_id = self.current_track_id is not None
         have_lineage = self.lineage is not None
-        init_sum = sum((have_current_track_id, have_lineage))
-        if init_sum == 2:
+        have_committed_lineages = self.committed_lineages is not None
+        have_tracking_widget = self.tracking_widget is not None
+        init_sum = sum((have_current_track_id, have_lineage, have_committed_lineages, have_tracking_widget))
+        if init_sum == 4:
             return True
         elif init_sum == 0:
             return False
         else:
             raise RuntimeError(
-                f"Invalid AnnotatorState: {init_sum} / 2 parts of the state "
+                f"Invalid AnnotatorState: {init_sum} / 4 parts of the state "
```

Comment:
If I saw this error message, I would still be a bit confused about what I needed to change to make things work again. Perhaps it could also tell users exactly which parts are not initialized correctly (out of `have_current_track_id, have_lineage`, `have_committed_lineages`, `have_tracking_widget`)

Reply:
I have updated the error messages to print the variables that are actually not initialized.